src/main/java/nl/tudelft/contextproject/script/UpdateTask.java

```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.contextproject.script;
+
+import nl.tudelft.contextproject.ContextTFP;
+
+import java.util.TimerTask;
+
+public class UpdateTask extends TimerTask {
```

Comment:
I think it would actually improve the readability of the code to use a lambda statement. That way you don't have to open the declaration of an UpdateTask in order to have an idea what that statement of code does. I can fix it for you if you don't feel like doing the effort yourself.